src/hotspot/share/classfile/verifier.cpp

```diff
@@ -105,11 +105,13 @@ static verify_byte_codes_fn_t verify_byte_codes_fn() {
 
 // Methods in Verifier
 
+// This method determines whether we run the verifier and class file format checking code.
 bool Verifier::should_verify_for(oop class_loader) {
   return class_loader == nullptr ?
     BytecodeVerificationLocal : BytecodeVerificationRemote;
 }
 
+// This method determines whether we allow package access in access checks in reflection.
```

Comment:
I added "This method" because the two methods right next to each other look the same but are different.  I want to say this one does one thing and this other one does something subtly different.  And it makes it a complete sentence.